### PR TITLE
Publishing to GitHub

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Added `release.yml`

### CODE REVIEW

File: .github/workflows/publish-macosArm64.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+env:
+  GRADLE_OPTS: -Dorg.gradle.daemon=true -Dorg.gradle.parallel=true -Dorg.gradle.welcome=never
+  GPG_SEC: ${{ secrets.PGP_SEC }}
+  GPG_PASSWORD: ${{ secrets.PGP_PASSWORD }}
+
+jobs:
+  publish:
+    name: 'Publish macosArm64'
+    runs-on: [self-hosted, macOS, ARM64]
```
Comment (nulls) [inline .github/workflows/publish-macosArm64.yml]: self-hosted?

File: .github/workflows/publish-macosArm64.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: 'Publish macosArm64'
```
Comment (nulls) [inline .github/workflows/publish-macosArm64.yml]: why do add it as a separate workflow? Maybe publish all targets in one workflow and use matrix?

File: .github/workflows/publish-macosArm64.yml (hunk 1 above)
Comment (sanyavertolet) [reply .github/workflows/publish-macosArm64.yml]: Discussed with Andrey K offline, it seems that there is no need for an extra self-hosted runner as macosX86 is able to build Kotlin for macosArm64

Will remove an extra workflow

Comment (sanyavertolet) [reply .github/workflows/publish-macosArm64.yml]: Removed self-hosted runner

File: .github/workflows/publish-macosArm64.yml (hunk 2 above)
Comment (sanyavertolet) [reply .github/workflows/publish-macosArm64.yml]: It's not there anymore

**Review by nulls — APPROVED** [review]

**Pull request merged**